build.gradle

```diff
@@ -45,17 +45,18 @@ dependencies {
     compile 'org.apache.commons:commons-lang3:3.11'
     compile 'commons-io:commons-io:2.8.0'
     compile 'org.jsoup:jsoup:1.13.1'
-    compile 'com.github.kwebio:shoebox:0.4.12'
+    compile 'com.github.kwebio:shoebox:0.4.5'
```

Comment:
Current shoebox version is 0.4.35 (I had to do a bunch of releases while trying to fix that problem :( ).

Reply:
This was a typo. I'm surprised Kweb still built and ran when asking for a non existent version of Shoebox.